sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -768,8 +770,11 @@ def test_multinomial():
     assert multinomial(sqrt(11)*I, 3) == -10
 
     assert multinomial(n, -2) == binomial(n - 2, -2)
-    assert multinomial(n, -pi).rewrite(gamma
-        ) == gamma(n - pi + 1)/(gamma(1 - pi)*gamma(n + 1)), multinomial(n, -pi)
+    # don't know if n is -1 (in which case the right answer
+    # would be obtained -- 0 -- but left in symbolic form
+    # the denominator term `gamma(n + 1)` might cancel with
+    # the same term in the numerator of a multiplicative factor)
```

Comment:
if n is 0 the expr works too
`gamma(0 - pi + 1)/(gamma(1 - pi)*gamma(0 + 1))`
no infinitives
`gamma(1 - pi)` terms cancel out

Reply:
Are you giving justification for why the result should be returned? If so, the justification for not doing so still holds: since n is an integer it could be -1 and that will create a zoo in the denominator. That will give us a correct result if this expression is never multipled by `gamma(n + 1)`. But since we don't have control of that we don't allow the rewrite...or else we are back where we started with the failing Sum.